lock-api/src/main/java/com/palantir/lock/v2/LeasableStartIdentifiedAtlasDbTransactionResponse.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.lock.v2;
+
+import org.immutables.value.Value;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+@Value.Immutable
+@JsonSerialize(as = ImmutableLeasableStartIdentifiedAtlasDbTransactionResponse.class)
+@JsonDeserialize(as = ImmutableLeasableStartIdentifiedAtlasDbTransactionResponse.class)
+public interface LeasableStartIdentifiedAtlasDbTransactionResponse {
+    @Value.Parameter
+    StartIdentifiedAtlasDbTransactionResponse getStartTransactionResponse();
```

Comment:
I wouldn't reference the old response here (it sets a dubious precedent around wrapping; the V10 response shouldn't contain all V9, V8, ... responses. Also, it's a bunch of JSON bloat (startTransactionResponse is like 30 bytes of text. I'd prefer just listing the fields that show up.